ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/SignatureIntegrationForm.tsx

```diff
@@ -257,6 +310,215 @@ function SignatureIntegrationForm({
+                                                                        <TextArea
+                                                                            autoResize
+                                                                            resizeOrientation="vertical"
+                                                                            type="text"
+                                                                            id={`cosignCertificates[${index}].certificatePemEnc`}
+                                                                            value={
+                                                                                values
+                                                                                    .cosignCertificates[
+                                                                                    index
+                                                                                ]
+                                                                                    .certificatePemEnc ||
+                                                                                ''
+                                                                            }
+                                                                            onChange={(
+                                                                                event,
+                                                                                value
+                                                                            ) =>
+                                                                                onChange(
+                                                                                    value,
+                                                                                    event
+                                                                                )
+                                                                            }
+                                                                            onBlur={handleBlur}
+                                                                            isDisabled={!isEditable}
+                                                                        />
+                                                                    </FormLabelGroup>
+                                                                </FlexItem>
+                                                            </Flex>
+                                                            {isEditable && (
+                                                                <FlexItem>
+                                                                    <Button
+                                                                        variant="plain"
+                                                                        aria-label="Delete header key/value pair"
```

Comment:
Aha, this looks like copy paste missed in original review:
https://github.com/stackrox/stackrox/blob/master/ui/apps/platform/src/Containers/Integrations/IntegrationForm/Forms/GenericWebhookIntegrationForm.tsx#L364

I will leave to your judgment whether to rewrite here and above.

Reply:
Transitive copy paste errors 😨